functions.go

```diff
@@ -108,3 +158,79 @@ func (s *FunctionsServiceOp) DeleteNamespace(ctx context.Context, namespace stri
 	}
 	return resp, nil
 }
+
+// Gets a list of triggers
```

Comment:
idiomatic go style for documentation would start with the function name first. most go linters would fail without it.

Eg.

```
// ListNamespaces lists function namespaces for a user.
```